src/api/timezones/timezones.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { paths } from '../types/generated-schema';
+
+const TIMEZONES_ENDPOINT = '/api/v1/i18n/timeZones';
+
+type ListTimeZoneNamesResponsePayload =
+  paths[typeof TIMEZONES_ENDPOINT]['get']['responses'][200]['content']['application/json'];
+
+export type TimeZone = ListTimeZoneNamesResponsePayload['timeZones'][0];
+
+export const getTimeZones = async (): Promise<TimeZone[]> => {
+  const response: ListTimeZoneNamesResponsePayload = (await axios.get(TIMEZONES_ENDPOINT)).data;
+  return response.timeZones;
```

Comment:
should we do the try catch and populate with requestSucceeded etc.. just like we do other API?
like here in updateAccession2 https://github.com/terraware/terraware-web/blob/main/src/api/accessions2/accession.ts
fwiw, looks like we aren't doing try/catch for getAccession2 but unrelated to this PR

Reply:
Right, I just used getAccession2 as guide and forgot about try/catch